evil-collection.el

```diff
@@ -38,16 +38,30 @@
 (require 'evil)
 
 (defvar evil-want-integration)
-(if (featurep 'evil-integration)
-    (if evil-want-integration
+(defvar evil-want-keybinding)
+(if (featurep 'evil-want-keybinding)
+    (if evil-want-keybinding
         (display-warning
          '(evil-collection)
-         "Make sure to set `evil-want-integration' to nil before loading evil \
-or evil-collection.")
+         "Make sure to set `evil-want-keybinding' to nil before loading evil \
+or evil-collection.\
+\n
+See https://github.com/emacs-evil/evil-collection/issues/60 for more details. ")
```

Comment:
Spurious trailing space.
Also why the `\`?  I think
```
or evil-collection.

See...
```
would be more readable.

Reply:
The formatting was better for me with that^.